src/simplewallet/simplewallet.cpp

```diff
@@ -5059,7 +4984,7 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
   // prompt is there is no payment id and confirmation is required
   if (!payment_id_seen && m_wallet->confirm_missing_payment_id() && dsts.size() > num_subaddresses)
   {
-     std::string accepted = input_line(tr("No payment id is included with this transaction. Is this okay?  (Y/Yes/N/No): "));
+     std::string accepted = input_line(tr("No payment id is included with this transaction: ask the recipient to use subaddresses instead. Continue anyway?  (Y/Yes/N/No): "));
```

Comment:
Is this a useful string? How about, if sent to a standard address (not a subaddress): `There is no easy way for exchanges and large merchants to identify that this transaction came from you. Request a subaddress in these cases. Continue anyway?`

Reply:
Thanks @SamsungGalaxyPlayer for reviewing. I've updated messages as recommended.